.github/workflows/publish.yml

```diff
@@ -0,0 +1,99 @@
+name: Publish API docs
+
+# on:
+#   push:
+#     branches:
+#     - main
+#   pull_request:
+on:  [push, pull_request]
```

Comment:
You probably want to change this back, so that docs are not published from unfinished PRs.

Reply:
Totally! Just haven't quite settled everything yet. Planning on releasing this after 25.1 is out, which I think will be on the 13th.